os/registry-authentication.md

```diff
@@ -0,0 +1,199 @@
+# Using authentication for a registry
+
+To pull images behind an authentication wall you need to setup some form of authentication with your image registry. These are either [quay.io][quay-site] or [hub.docker.com][docker-hub-site] or a self-hosted `registry.example.io/v0`.
+
+This document outlines the preferred method of authentication with an image registry like [quay.io][quay-site] and [hub.docker.com][docker-hub-site] and how setup registry auth for Docker, Kubernetes, rkt, and Mezos.
+
+## Using a Quay robot for registry auth
+
+The preferred way to setup registry authentication with [quay.io][quay-site] is to use a [Quay Robot][quay-robot]. This essentially acts as a login-token wth some nice features including:
+
+- Pre-populated repository-auth configuration files.
+- Credentials are limited to specific repositories.
+- Choose from read, write, or admin privileges.
+- Token regeneration.
+
+![Quay Robot settings][quay-bot-img]
```

Comment:
It'd probably be a good idea to crop this image so it doesn't contain your username in it